### PR TITLE
[WIP] New Package: jekyll-3.1.2

### PR DESCRIPTION
**Pull request by TheDoctorsLife**
First time making a rupy module in void.

### CODE REVIEW

**Comment by Vaelatern** [comment]
Perhaps .gem should be on `skip_extraction="filename"`?

**Comment by TheDoctorsLife** [comment]
That worked, stuck on a new error now though.

**Comment by ebfe** [comment]
create_wrksrc=yes

**Comment by TheDoctorsLife** [comment]
That helped, though I'm stuck on a new error

File: srcpkgs/jekyll/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+depends="ruby nodejs"
+homepage="https://github.com/jekyll/jekyll"
+create_wrksrc=yes
+skip_extraction="${pkgname}-${version}.gem"
+distfiles="https://rubygems.org/downloads/${pkgname}-${version}.gem"
+checksum="6788a948239e7f1e4ed297083485059ee31970dfdd5c2626d30b39e8cf39a28c"
+
+do_install() {
+local _gemdir="$(ruby -e'puts Gem.default_dir')"
+gem install --no-user-install -i "${_gemdir}" -n "${DESTDIR}/usr/bin" ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${pkgname}-${version}.gem
```
Comment (Vaelatern) [inline srcpkgs/jekyll/template]: Above this line: `mkdir -p ${DESTDIR}/usr/bin`

**Comment by TheDoctorsLife** [comment]
It works!

**Comment by TheDoctorsLife** [comment]
I was asked in the IRC to split out the deps from the jekyll package, please don't merge for now.

**Comment by xtraeme** [comment]
This won't be accepted. You are just packaging the ruby gems directly... you can accomplish the same and for your user with the `gem` application directly.

Thanks!

**Pull request closed without merging**